gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Gherkin/CommonSteps.cs

```diff
@@ -127,20 +127,20 @@ public void TranslateTraversal(string traversalText)
                 throw new InvalidOperationException("g should be a traversal source");
             }
 
-            if (ScenarioData.CurrentFeature.Tags.Select(t => t.Name).ToList().Contains("@GraphComputerOnly"))
+            if (ScenarioData.CurrentFeature!.Tags.Select(t => t.Name).ToList().Contains("@GraphComputerOnly"))
```

Comment:
why is there `.ToList()` needed? 
```suggestion
            if (ScenarioData.CurrentFeature!.Tags.Any(t => t.Name == "@GraphComputerOnly"))
```

Reply:
It shouldn't be needed. I updated the commit to include your suggestion.